src/os/osexec.go

```diff
@@ -0,0 +1,56 @@
+//go:build linux
+// +build linux
```

Comment:
You only need `//go:build linux`, the old `// +build` lines are deprecated.
Also, you'll probably want `//go:build linux && !baremetal`.